CHANGELOG.md

```diff
@@ -11,5 +11,7 @@
 - Fixed: [#5502](https://github.com/ethereum/aleth/pull/5502) Fix Discovery terminating prematurely because of race condition.
 - Fixed: [#5452](https://github.com/ethereum/aleth/pull/5452) Correctly handle Discovery messages when the peer changes public key.
 - Fixed: [#5519](https://github.com/ethereum/aleth/pull/5519) Correctly handle Discovery messages with known public key but unknown endpoint.
+- Fixed: [#5523](https://github.com/ethereum/aleth/pull/5523) Fix syncing terminating prematurely because of race condition.
+- Fixed: [#5533](https://github.com/ethereum/aleth/pull/5533) Remove unnecessary ioservice polling on network shutdown.
```

Comment:
I would skip this, too low-level for the user and it's something that we introduced and then quickly fixed inside one release.

Reply:
Addressed.